### DISCUSSION TITLE
Dual parameterization for SVGP

### DISCUSSION THREAD

**Discussion by Turakar** [cornellius-gp/gpytorch#2405] [Ideas]
I currently do not have the time to understand this paper completely, but I want to raise your attention to it:

[Dual Parameterization of Sparse Variational Gaussian Processes](https://proceedings.neurips.cc/paper_files/paper/2021/hash/5fcc629edc0cfa360016263112fe8058-Abstract.html)

On a first read, it seems like there's plenty of optimization potential in how we do variational inference in GPyTorch, both in accuracy and performance.

**Comment by gpleiss** [comment]
It's a good paper, and we'd definitely be interested in adding it if someone is willing to implement and put up a PR.
